### PR TITLE
update redactor spec for v0.9.37

### PR DESCRIPTION
**Pull request by laverya**
https://github.com/replicatedhq/troubleshoot.sh/pull/114

### CODE REVIEW

File: redact/source/reference.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: "Reference"
 ---
```
Comment (marccampbell) [inline redact/source/reference.md:4]: This document should be titled `API Reference` to match Collectors and Analyzers

File: redact/gatsby-config.js [hunk 2]
```diff
@@ -23,11 +23,15 @@ module.exports = {
...
           null: [
             "index",
-            "reference",
           ],
-          Redactors: [
+          "Default Redactors": [
             "database-connection-strings",
             "aws-credentials",
+            "generic-connection-strings",
+            "passwords-and-tokens",
```
Comment (marccampbell) [inline redact/gatsby-config.js:31]: Should IP Addresses be separated into it's own item?
Can we split passwords and API tokens into 2?
I think the "generic connection strings" looks more like "http and ftp endpoints with usernames and passwords".  What name could we use to make this more specific, "generic connection strings" doesn't make sense to me here.

File: redact/gatsby-config.js [hunk 3]
```diff
@@ -23,11 +23,15 @@ module.exports = {
         sidebarCategories: {
           null: [
             "index",
-            "reference",
           ],
-          Redactors: [
+          "Default Redactors": [
```
Comment (marccampbell) [inline redact/gatsby-config.js:27]: Let's keep this as `Redactors`. We will be extending this section to match Collectors and Analyzers where the list will be the built-in functions that can be extended.

File: redact/source/reference.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ description: "Reference"
...
 
 Redactors are YAML specifications that define which data to remove when generating a support bundle.
+Redactors are currently in alpha.
```
Comment (marccampbell) [inline redact/source/reference.md:7]: Redactors are currently an alpha feature.

File: redact/gatsby-config.js (hunk 2 above)
Comment (laverya) [reply redact/gatsby-config.js:31]: This isn't what shows up in the list - this is the filepath

here's a screenshot:
image

**Pull request merged**